### PR TITLE
releasing instructions updates

### PR DESCRIPTION
**Pull request by nader-ziada**
# Changes

 - Made some changes to the release instructions to try to make it more clear

/kind documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,434 +66,6 @@ This roster is seeded with approvers from the Client workgroups.
...
-
-1. Send a PR like [this one](https://github.com/knative/community/pull/619) to
-grant ACLs for the next release leads, and to remove yourself from the rotation.
-Include the next release leads in the PR as a reminder.
-
-1. Send a PR like [this one](https://github.com/knative-sandbox/knobots/pull/152/files) to
-bump knobots auto release workflow to the next release.
+# Release Process
 
+[Instructions](./releasing.md) on how to release a `knative.dev` repository.
```
Comment (kvmware) [inline README.md:71]: Not a fan of the redirect here. Why not just have everything in the root?

**Comment by kvmware** [comment]
Other than the redirect I like what I see. Makes actual sense and one does not have to jump around. Awesome work :+1:

File: README.md (hunk 1 above)
Comment (gabo1208) [reply README.md:71]: For me it makes sense, you can have a concrete guide for the release process without all the roster and release versions distracting

**Comment by gabo1208** [comment]
Nice job @nader-ziada 👍🏼 ! This is ok for me (if the checks doesn't start try closing and reopening the PR)

File: releasing.md [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+  - Replace the `sha256` checksums with the values from the [client release page](https://github.com/knative/client/releases). The checksums have been released, too (e.g. [checksums.txt](https://github.com/knative/client/releases/download/v0.22.0/checksums.txt))
+
+Create a PR and merge the changes. Prow is not enabled for the homebrew repo, so the merge needs to be performed manually.
+
+#### homebrew-kn-plugins
+
+Similar to the client repo, the [client plugin's Homebrew repo](https://github.com/knative-sandbox/homebrew-kn-plugins) needs to be updated
+for the the plugins supported after their repos have successfully created a release.
+
+Currently the following plugins are available with their own formulas:
```
Comment (psschwei) [inline releasing.md:224]: We should also add `kn-plugin-event` to this list

File: README.md [hunk 3]
```diff
@@ -66,434 +66,6 @@ This roster is seeded with approvers from the Client workgroups.
 
 ---
 
-# Instructions
```
Comment (psschwei) [inline README.md:69]: Since we're making changes here, we may want to drop @julz and @ZhiminXiang from the serving roster

Comment (nader-ziada) [reply README.md:69]: done and also added myself as a volunteer, (can I just add myself?)

**Comment by nader-ziada** [comment]
@dprotaso @psschwei let me know what changes (if any) you all want to do to the serving release lead schedule coming up

File: README.md (hunk 3 above)
Comment (dprotaso) [reply README.md:69]: > (can I just add myself?)

Yeah

File: releasing.md [hunk 4]
```diff
@@ -0,0 +1,393 @@
+
```
Comment (dprotaso) [inline releasing.md:1]: nit: make the file name all caps?

File: releasing.md [hunk 5]
```diff
@@ -0,0 +1,393 @@
...
+### 1. Cut release branches of supporting repos
+
+The supporting repos are the base repos where we have common code and common
+scripts. For these repos (except **hack**), we follow the same release process as
+explained in [release a repository](#release-a-repository), but no prow job is
+executed, hence no git tag and Github release are produced.
+
+```
+NOTE: The supporting repos must be released in the correct order, so you need to check if the [dependencies should be updated](#aligning-the-dependencies) after each repo release. 
+```
```
Comment (dprotaso) [inline releasing.md:56]: This doesn't render properly

https://github.com/knative/release/blob/91e1c5bd167b724e6248657d3d19c38310cca6ba/releasing.md#1-cut-release-branches-of-supporting-repos

Comment (nader-ziada) [reply releasing.md:56]: are you using Safari? I'll change it but wondering

File: releasing.md (hunk 4 above)
Comment (nader-ziada) [reply releasing.md]: ok

**Comment by psschwei** [comment]
> > > @dprotaso @psschwei let me know what changes (if any) you all want to do to the serving release lead schedule coming up
> > 
> > 
> > I can swap in for v1.6
> 
> I can take 1.8, but don't want 1.9 since I might be away

Sorry, I meant I can do 1.6 and 1.9 (swap probably wasn't the right word there, should've swapped it for something else.... I'll see myself out now :) )

**Comment by dprotaso** [comment]
I'm good with whatever - will re-adjust if anything changes

**Comment by nader-ziada** [comment]
I think PR is ready now if anybody wants to approve

**Review by psschwei — COMMENTED** [review]
/lgtm

**Pull request merged**